prusti-tests/tests/cargo_verify/veribetrfs/src/base/vector.rs

```diff
@@ -0,0 +1,42 @@
+use prusti_contracts::*;
+
+pub struct Vector<T>(Vec<T>);
+
+impl<T> Vector<T> {
```

Comment:
You can find examples showing how to add specs to Rust's `Vec` [here](https://github.com/viperproject/prusti-dev/tree/e51e0c21b5f0ec5ad7e7ca9b3b8eda322965c0cb/prusti-tests/tests/verify/pass/extern-spec). However, I am not sure whether `index` and `index_mut` are properly supported.